src/Patches/PlayerCharacterPatches.cs

```diff
@@ -148,6 +149,10 @@ public static void PlayerCharacter_Update_PostfixPatch(PlayerCharacter __instanc
                 PlayerCharacter.kStopDropRollDistancePerSecondThreshold = 10;
             }
 
+            if (BaldFox) {
+                __instance.gameObject.transform.Find("fox hair").gameObject.active = false;
+            }
```

Comment:
```suggestion
                __instance.gameObject.transform.Find("fox hair").GetComponent<Renderer>().enabled = BaldFox;
```
Other things reference that gameobject so we shouldn't disable the entire thing